docs/release-notes/2092.fix.md

```diff
@@ -0,0 +1 @@
+Writing of views of {class}`~anndata.AnnData` in backed mode {user}`ilan-gold`
```

Comment:
This sentence describes an action, but not how the fix relates to that action!

Reply:
Yes, sorry, I was thinking "the note's title is `fix` and the fix was that this action literally never worked"